### ISSUE TITLE
s25update versions differ for different OS

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#916]
Currently windows downloads 20181108 but mac downloads 20181101. Should be the same for any OS.

**Comment by Flamefire** [comment]
Probably the same as #915: Build server fails on some builds.

**Comment by Flamefire** [comment]
@Flow86 Something is wrong with the build server. Spurious failures with `After 10s process did not stop`

**Comment by stefson** [comment]
from my selfcompiled binary: 

```
~ $ s25client 
Return To The Roots v20181111-03556d14626922cf9e11d95351fe20f91aebda34
Compiled with Clang (6.0.1) for Linux, Unix Environment 64 Bit
```

so it's likely the build server! ;-)

**Comment by Flamefire** [comment]
Fixed with  06a6650
